nilearn/interfaces/fmriprep/load_confounds.py

```diff
@@ -403,19 +412,98 @@ def _load_confounds_for_single_image_file(
     flag_full_aroma = ("ica_aroma" in strategy) and (
         kwargs.get("ica_aroma") == "full"
     )
-
+    # Check for tedana
+    flag_tedana = ("tedana" in strategy) and (kwargs.get("tedana") == "rejected")
     confounds_file = get_confounds_file(
-        image_file, flag_full_aroma=flag_full_aroma
+        image_file, flag_full_aroma=flag_full_aroma, flag_tedana=flag_tedana
     )
-    confounds_json_file = get_json(confounds_file)
+    confounds_json_file = get_json(confounds_file, flag_tedana=flag_tedana)
+
+    if flag_tedana:
+        # If tedana is in the strategy, we need to load the tedana confounds and apply the tedana strategy.
+        # file instead of the fmriprep confounds file
+        return _load_tedana_confounds_file(
+            confounds_file=confounds_file,
+            strategy=strategy,
+            demean=demean,
+            confounds_json_file=confounds_json_file,
+            **kwargs,
+        )
+    else:
+        return _load_single_confounds_file(
+            confounds_file=confounds_file,
+            strategy=strategy,
+            demean=demean,
+            confounds_json_file=confounds_json_file,
+            **kwargs,
+        )
```

Comment:
else not needed as we return early

```suggestion
    return _load_single_confounds_file(
            confounds_file=confounds_file,
            strategy=strategy,
            demean=demean,
            confounds_json_file=confounds_json_file,
            **kwargs,
        )
```